multiversx_sdk_cli/cli_accounts.py

```diff
@@ -32,6 +37,8 @@ def _add_address_arg(sub: Any):
 
 
 def get_account(args: Any):
+    omitted_fields = cli_shared.parse_omit_fields_arg(args)
```

Comment:
Though, do we still need this command, or can we drop it (since people can simply get the data from the public API)?

Reply:
as decided, we've dropped this command.